cli/command/manifest/annotate_test.go

```diff
@@ -0,0 +1,59 @@
+package manifest
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/docker/cli/cli/internal/test"
+	"github.com/docker/docker/pkg/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewAnnotateCommand(t *testing.T) {
+	testCases := []struct {
+		name		string
+		args		[]string
+		expectedError	string
+	}{
+		{
+			name:		"wrong-args",
+			args:		[]string{},
+			expectedError:	"\"annotate\" requires exactly 2 argument(s).",
+		},
+	}
+	for _, tc := range testCases {
+		buf := new(bytes.Buffer)
+		cmd := newAnnotateCommand(test.NewFakeCli(&fakeClient{}, buf))
```

Comment:
Just a heads-up that this has changed on master. `NewFakeCli()` no longer takes a second arg. It uses a `bytes.Buffer` by default now, which you can access using the `OutBuffer()` method.